standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -5326,6 +5355,23 @@ private void preDropStorageDescriptor(MStorageDescriptor msd) {
     removeUnusedColumnDescriptor(mcd);
   }
 
+  /**
+   * Get a list of storage descriptors that reference a particular Column Descriptor
+   * @param oldCD the column descriptor to get storage descriptors for
```

Comment:
If we keep the query parameter, please document it

Reply:
Will add.